### PR TITLE
First pass at login UI

### PR DESCRIPTION
**Pull request by ivoryfitz**
Just a short first pass at getting it away from "vanilla html". Can clean it up more later if need be. There's a bit of redirection within the Client FSM that I thought about changing (on successful account creation, probably should go back to the login panel) but I didn't want to touch more than I had to.

Not sure the best way to "review" UI changes like this. Should I attach some screenshots or do a short screen capture? Unsure. Let me know what you think.

### CODE REVIEW

**Comment by jonathaneeckhout** [comment]
Thanks for the pull request. I will take it in and have a look. I still need to write down contribution guides so I will now check it manually.

**Comment by jonathaneeckhout** [comment]
Overall a great improvement, thanks. I like the transition animations.

Here are 2 comments I have regarding the ui:
- The font-size for the line edit fields is a bit too small to clearly read. Maybe make it a bit bigger for readability?
- This is a personal opinion but I am not convinced of the red color of the buttons. WDYT?

**Comment by ivoryfitz** [comment]
Okay let me see if I can up the size on the line edits. I agree they seemed small, but I wasn't sure if that was just my eyes being "old" hahaha. Honestly, I think I'm missing something with the scaling over all for the viewport size I need to watch a few more videos but for now I'll just try to up the size on those line edits.

Red button seemed off, but that was a similar coloration to the "Embark" button on Darkest Dungeon. Let me change the text to an eggshell/off-white instead of red, I bet it will look a lot better and more readable. I'll update after church a bit later today. Thanks for the review!

**Comment by ivoryfitz** [comment]
Addressed feedback and made a few other QOL changes. Hopefully this is a bit better. I'm going to be checking out https://www.youtube.com/watch?v=1_OFJLyqlXI at some point. Looks like a pretty comprehensive overview of UI in Godot so I might learn some more tricks to apply later down the line.

Let me know if you want something different here!

**Comment by jonathaneeckhout** [comment]
When creating a new account, I get the following error message:

E 0:00:16:0524   client.gd:88 @ alertbox(): In Object of type 'AcceptDialog': Attempt to connect nonexistent signal 'modal_closed' to callable 'AcceptDialog::queue_free'.
  <C++ Error>    Condition "!signal_is_valid" is true. Returning: ERR_INVALID_PARAMETER
  <C++ Source>   core/object/object.cpp:1280 @ connect()
  <Stack Trace>  client.gd:88 @ alertbox()
                 Client.gd:147 @ _handle_create_account()
                 account.gd:18 @ create_account_response()

File: assets/fonts/fenwick-woodtype/1001fonts-fenwick-woodtype-eula.txt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ - t-shirts, books, apparel that will be sold for money
+ - flyers, posters for events that charge admission
+ - freelance graphic design work
+ - anything that will generate direct or indirect income
+
+3. Modification
+Fenwick Woodtype may not be modified, altered, adapted or built upon without written permission by its respective author. This pertains all files within the downloadable font zip-file.
+
+4. Distribution
+While Fenwick Woodtype may freely be copied and passed along to other individuals for private use as its original downloadable zip-file, it may not be sold or published without written permission by its respective author.
```
[the remark this answers is not in the document]
Comment (ivoryfitz) [reply assets/fonts/fenwick-woodtype/1001fonts-fenwick-woodtype-eula.txt:28]: Addressing this in Discord with you

[the remark this answers is not in the document]
Comment (jonathaneeckhout) [reply assets/fonts/fenwick-woodtype/1001fonts-fenwick-woodtype-eula.txt:28]: As we will not explicitly redistribute this font I think we're fine here. Resolving review

**Comment by ivoryfitz** [comment]
@jonathaneeckhout As far as the singleton goes, I went ahead and added one, but I'm not too familiar with how it should be done within Godot. This is my attempt, if you want directory structure different or different naming, I don't mind, just let me know what you prefer and I can affect change. This is what I've got for now though.

**Pull request merged**